source/java/spring-boot-web/tutorial.html

```diff
@@ -0,0 +1,845 @@
+<p>The <code class="docutils literal"><span class="pre">configure</span></code> method of <code class="docutils literal"><span class="pre">StormpathWebSecurityAutoConfiguration</span></code> does some housekeeping to ensure that all of the Stormpath
+views (such as /login) remain available and then it calls your <code class="docutils literal"><span class="pre">doConfigure</span></code> method.</p>
+<p>In this case, we are expressing that anyone trying to get to <code class="docutils literal"><span class="pre">/me</span></code> should be fully authenticated.</p>
+<p>For more on <code class="docutils literal"><span class="pre">HttpSecurity</span></code> with Spring Security, look <a class="reference external" href="http://docs.spring.io/spring-security/site/docs/current/reference/htmlsingle/#jc-httpsecurity">here</a>.</p>
+<p>The new method to handle <code class="docutils literal"><span class="pre">me</span></code> requests in our <code class="docutils literal"><span class="pre">HelloController</span></code> does not call out any other authorizaton requirements. Anyone logged in will be able to
+access <code class="docutils literal"><span class="pre">/me</span></code>. Furthermore, anyone NOT logged in trying to access <code class="docutils literal"><span class="pre">/me</span></code> will automatically be redirected to the <code class="docutils literal"><span class="pre">/login</span></code> view.</p>
+<div class="highlight-java"><table class="highlighttable"><tr><td class="linenos"><div class="linenodiv"><pre> 1
+ 2
+ 3
+ 4
+ 5
+ 6
+ 7
+ 8
+ 9
+10
+11
+12</pre></div></td><td class="code"><div class="highlight"><pre><span class="nd">@Controller</span>
+<span class="kd">public</span> <span class="kd">class</span> <span class="nc">HelloController</span> <span class="o">{</span>
+
+    <span class="o">...</span>
+
+    <span class="nd">@RequestMapping</span><span class="o">(</span><span class="s">&quot;/me&quot;</span><span class="o">)</span>
+    <span class="n">String</span> <span class="nf">me</span><span class="o">()</span> <span class="o">{</span>
+        <span class="k">return</span> <span class="s">&quot;me&quot;</span><span class="o">;</span>
+    <span class="o">}</span>
+
+    <span class="o">...</span>
+<span class="o">}</span>
+</pre></div>
+</td></tr></table></div>
+<p>Try it out. Launch the application as before, and then browse to: <code class="docutils literal"><span class="pre">http://localhost:8080/me</span></code>. You will be redirected to the <code class="docutils literal"><span class="pre">/login</span></code>
+and then after you login, you will automatically be brought back to <code class="docutils literal"><span class="pre">/me</span></code>. That&#8217;s the StormPath magic at work!</p>
```

Comment:
"...and then after you login to a valid Stormpath Account, you will be automatically brought..."